opentracing-spring-web-autoconfigure/src/main/java/io/opentracing/contrib/spring/web/autoconfig/ServerTracingAutoConfiguration.java

```diff
@@ -25,11 +29,26 @@
 public class ServerTracingAutoConfiguration {
     private static final Logger log = Logger.getLogger(ServerTracingAutoConfiguration.class.getName());
 
+    @Autowired(required=false)
+    @Qualifier("opentracing.http.skipPattern")
+    private Set<String> skipPatterns;
```

Comment:
`WebTracingConfiguration` should hold all configuration related to this package. Could we stick to that? 

Other auto-configs should be able to adjust this value if necessary.